mmv1/third_party/terraform/tests/data_source_backup_dr_management_server_test.go

```diff
@@ -0,0 +1,71 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccDataSourceGoogleBackupDRManagementServer_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceGoogleBackupDRManagementServer_basic(context),
+				Check: resource.ComposeTestCheckFunc(
+					acctest.CheckDataSourceStateMatchesResourceState("data.google_cloud_backup_dr_management_server.foo", "google_cloud_backup_dr_management_server.foo"),
+				),
+			},
+		},
+	})
+}
+
+
+func testAccDataSourceGoogleBackupDRManagementServer_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+	resource "google_compute_network" "default" {
+		provider = google-beta
+		name = "vpc-network"
+	  }
+	  
+	  resource "google_compute_global_address" "private_ip_address" {
+		provider = google-beta
+		name          = "vpc-network"
+		address_type  = "INTERNAL"
+		purpose       = "VPC_PEERING"
+		prefix_length = 20
+		network       = google_compute_network.default.id
+	  }
+	  
+	  resource "google_service_networking_connection" "default" {
```

Comment:
service networking connection creates some tenant resource that can't be cleaned up by our team. So we need to bootstrap network to reuse them. Would you mind following this [example](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/alloydb/resource_alloydb_backup_test.go#L14) to add bootstrapped network to your test? You'll need to change `alloydb-backup-update` in `acctest.BootstrapSharedTestNetwork(t, "alloydb-backup-update")` to a name, likely something related to the test name

Reply:
Added in latest commit